home/import_helpers.py

```diff
@@ -165,16 +165,36 @@ def extract_errors(data: dict[str | int, Any] | list[str]) -> dict[str, str]:
     return error_message
 
 
-def fix_rows(rows: Generator[dict[str, str], None, None]) -> Iterator[dict[str, str]]:
+def fix_rows(
+    rows: Generator[dict[str, str], None, None] | Iterator[dict[str | Any, Any]]
+) -> Iterator[dict[str, str | None]]:
     """
-    Fix keys for all rows by lowercasing and removing whitespace
+    Fix keys for all rows by lowercasing keys and removing whitespace from keys and values
     """
     for row in rows:
         yield fix_row(row)
 
 
-def fix_row(row: dict[str, str]) -> dict[str, str]:
-    return {k.lower().strip(): v.strip() if v else v for k, v in row.items()}
+def fix_row(row: dict[str, str]) -> dict[str, str | None]:
```

Comment:
If `_normalise_value()` accepts `str | None`, `fix_row()` should accept `dict[str, str | None]`.